### ISSUE TITLE
The return type for `createCable` is unresolved/any instead of Cable

### ISSUE THREAD

**Issue by kcrwfrd** [anycable/anycable-client#39]
<img width="957" alt="Screenshot 2024-04-25 at 2 26 57 PM" src="https://github.com/anycable/anycable-client/assets/504158/25fe5358-0079-4166-a3cf-0a6b1386c161">

This is causing type errors in our project when trying to use anycable.

I think you need to import all of the imports from `@anycable/core`, not export them.

```ts
import {
  CreateOptions,
  Cable,
  ActionCableConsumer,
  TokenRefresher
} from '@anycable/core'

// is this necessary?
export { CreateOptions, Cable, ActionCableConsumer, TokenRefresher }
```

**Comment by palkan** [comment]
Thanks for the report!

It looks like it's gonna be fixed by #38.

> I think you need to import all of the imports from @anycable/core, not export them.

We re-export types for usability (so, you can import all you need from the `@anycable/web` package without event knowing that `@anycable/core` exists and think about what to import from where).

**Comment by palkan** [comment]
Should be fixed in v0.9.0. Please, let me no if the problem still there, and we'll reopen it.
